src/object.rs

```diff
@@ -531,6 +531,15 @@ macro_rules! glib_object_wrapper {
             }
         }
 
+        #[doc(hidden)]
+        impl $crate::translate::FromGlibPtrBorrow<*const $ffi_name> for $name {
+            #[inline]
+            #[cfg_attr(feature = "cargo-clippy", allow(cast_ptr_alignment))]
+            unsafe fn from_glib_borrow(ptr: *const $ffi_name) -> Self {
+                $crate::translate::from_glib_borrow(ptr as *mut _)
```

Comment:
This could also be an explicit cast to `*mut $ffi_name`, but otherwise all good :) Please merge after changing that

Reply:
You order, I execute!

Reply:
No, it needs to be `as _`.

Reply:
Otherwise we get:

```
error[E0277]: the trait bound `object::ObjectRef: translate::FromGlibPtrBorrow<*mut gobject_ffi::GBinding>` is not satisfied
   --> src/object.rs:529:23
    |
529 |                   $name($crate::translate::from_glib_borrow(ptr as *mut $ffi_name),
    |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `translate::FromGlibPtrBorrow<*mut gobject_ffi::GBinding>` is not implemented for `object::ObjectRef`
    | 
   ::: src/gobject/auto/binding.rs:12:1
    |
12  | / glib_wrapper! {
13  | |     pub struct Binding(Object<ffi::GBinding, BindingClass>);
14  | |
15  | |     match fn {
16  | |         get_type => || ffi::g_binding_get_type(),
17  | |     }
18  | | }
    | |_- in this macro invocation
    |
    = help: the following implementations were found:
              <object::ObjectRef as translate::FromGlibPtrBorrow<*const gobject_ffi::GObject>>
              <object::ObjectRef as translate::FromGlibPtrBorrow<*mut gobject_ffi::GObject>>
note: required by `translate::from_glib_borrow`
   --> src/translate.rs:1052:1
    |
1052| pub unsafe fn from_glib_borrow<P: Ptr, T: FromGlibPtrBorrow<P>>(ptr: P) -> T {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0277]: the trait bound `object::ObjectRef: translate::FromGlibPtrBorrow<*mut gobject_ffi::GBinding>` is not satisfied
   --> src/object.rs:529:23
    |
529 |                   $name($crate::translate::from_glib_borrow(ptr as *mut $ffi_name),
    |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `translate::FromGlibPtrBorrow<*mut gobject_ffi::GBinding>` is not implemented for `object::ObjectRef`
    | 
   ::: src/gobject/auto/binding.rs:12:1
    |
12  | / glib_wrapper! {
13  | |     pub struct Binding(Object<ffi::GBinding, BindingClass>);
14  | |
15  | |     match fn {
16  | |         get_type => || ffi::g_binding_get_type(),
17  | |     }
18  | | }
    | |_- in this macro invocation
    |
    = help: the following implementations were found:
              <object::ObjectRef as translate::FromGlibPtrBorrow<*const gobject_ffi::GObject>>
              <object::ObjectRef as translate::FromGlibPtrBorrow<*mut gobject_ffi::GObject>>
note: required by `translate::from_glib_borrow`
   --> src/translate.rs:1052:1
    |
1052| pub unsafe fn from_glib_borrow<P: Ptr, T: FromGlibPtrBorrow<P>>(ptr: P) -> T {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error
```